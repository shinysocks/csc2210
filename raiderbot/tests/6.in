+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*                  ||
||                  ||
|+------------------+|
+--------------------+
19 9
NS
WWWWWWNW
swWWWWWWWWWW
NW
NE
W
W
