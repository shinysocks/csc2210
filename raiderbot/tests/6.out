Initial state: Robot at 19, 9 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*                  ||
||                  ||
|+------------------R|
+--------------------+
========================================
Executing NS
Robot at 19, 9 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*                  ||
||                  ||
|+------------------R|
+--------------------+
Executing WWWWWWNW
Robot at 12, 8 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*                  ||
||           R      ||
|+------------------+|
+--------------------+
Executing swWWWWWWWWWW
Robot at 1, 9 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*                  ||
||                  ||
|+R-----------------+|
+--------------------+
Executing NW
Robot at 0, 8 (0 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|*                  ||
|R                  ||
|+------------------+|
+--------------------+
Executing NE
Robot at 1, 7 (1 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
| R                 ||
||                  ||
|+------------------+|
+--------------------+
Executing W
Robot at 0, 7 (1 gold)
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|R                  ||
||                  ||
|+------------------+|
+--------------------+
Executing W

Error: robot with current status of Robot at 0, 7 (1 gold) broke while executing W
+--------------------+
|+------------------+|
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
||                  ||
|R                  ||
||                  ||
|+------------------+|
+--------------------+
========================================
